### PR TITLE
Create new db user in `bin/setup`

### PR DESCRIPTION
**Pull request by waldofouche**
Tried this PR locally, encountered this issue when running bin/setup for the first time:
```
Mysql2::Error::ConnectionError:Access denied for user 'validator_app'@'172.17.0.1' (using password: YES) 
```
Just created a simple user, just an idea no need to actually merge this PR

### CODE REVIEW

File: bin/setup [hunk 1]
```diff
@@ -20,6 +20,9 @@ if File.exist?('env.dist') && !File.exist?('.env')
   FileUtils.copy('env.dist', '.env')
 end
 
+puts("\n== Creating database user ==")
+system("mysql -e \"grant all on *.* to 'validator_app'@'localhost' identified by 'password';\"")
+
```
Comment (ritchiey) [inline bin/setup:25]: I think it's a good idea but I'm not sure it belongs in the Docker branch/PR given that those deployment targets don't use `bin/setup`.

I know I've got changes to that file in my PR as well but I'm thinking that I should probably extract those into a separate PR because they apply at least as much for OWO.

Comment (waldofouche) [reply bin/setup:25]: > those deployment targets don't use bin/setup

Ahh yep, I think I misunderstood how the docker one was currently set up.
I will just close this PR since it is based on top of your docker branch
